README.md

```diff
@@ -1,10 +1,11 @@
 # Legacy Fabric API
 
-Essential hooks for modding with Fabric.
+Essential hooks for modding on 1.7.10, 1.8.9 and 1.12.2.
 
-Fabric API is the library for essential hooks and interoperability mechanisms for Fabric mods. Examples include:
+Legacy Fabric API is the library for essential hooks and interoperability mechanisms for Legacy Fabric mods. Examples include:
 
-Exposing functionality that is useful but difficult to access for many mods such as particles, biomes and dimensions
-Adding events, hooks and APIs to improve interopability between mods.
+- Exposing functionality that is useful but difficult to access for many mods such as biomes and enchantments
+- Adding events, hooks, APIs and commonly used hacks to improve interoperability between mods such as resource loading, networking and gamerules
+- Adding APIs that help circumvent difficult-to-use minecraft APIs such as commands
 
-For support, consider joining the [Legacy Fabric discord server](https://legacyfabric.net/discord) or opening an issue.
+For support, consider joining the [Legacy Fabric discord server](https://discord.gg/emgF7bp), opening an issue, or creating a discussion.
```

Comment:
why change the discord link?

Reply:
I refactored Rewoven API, and the link in that was Rewoven's Discord.
The change there is unintentional, but also completely unimpactful as it's the same result anyway.

Reply:
Also finished.